.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: python
+python:
+- '3.6'
+- '3.5'
+- '3.4'
+
+matrix:
+  include:
+    - python: 3.6
+      sudo: required
+      dist: trusty
+      env:
+      - TOXENV=style
+      before_install:
+      - sudo sed -i 's/trusty/yakkety/' /etc/apt/sources.list
+      - sudo rm -rf /etc/apt/sources.list.d/*
```

Comment:
Line 1: It's changing the Ubuntu release code name in the APT sources list, so when I install `spellcheck`, the version from Yakkety is installed. (Trusty have one version in backports, but that one is old and reports some false errors.)

Line 2: Removing any 3rd party repositories available on Travis CI (not needed, proven to make trouble when doing Line 1).